VideoLocker/res/values/styles.xml

```diff
@@ -37,6 +37,7 @@
         <item name="android:activatedBackgroundIndicator">@drawable/activated_item_selector</item>
         <item name="android:windowAnimationStyle">@style/EnterExitAnimation.Activity</item>
         <item name="android:dropDownListViewStyle">@style/edX.Widget.SpinnerDropDownListView</item>
+        <item name="android:textDirection">locale</item>
```

Comment:
It falls back only if there is no "strong directional character”, which is an unlikely occurrence on most texts ;)
> On 11-Sep-2015, at 11:29 pm, Brian Guertin <notifications@github.com> wrote:
> 
> In VideoLocker/res/values/styles.xml <https://github.com/edx/edx-app-android/pull/358#discussion_r39301746>:
> 
> > @@ -37,6 +37,7 @@
> >          <item name="android:activatedBackgroundIndicator">@drawable/activated_item_selector</item>
> >          <item name="android:windowAnimationStyle">@style/EnterExitAnimation.Activity</item>
> >          <item name="android:dropDownListViewStyle">@style/edX.Widget.SpinnerDropDownListView</item>
> > +        <item name="android:textDirection">locale</item>
> I'm not sure why this is needed. The default firstStrong is supposed to fall back to the layout direction, which is already RTL
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/358/files#r39301746>.
>